plugins/node/opentelemetry-plugin-pg/src/enums.ts

```diff
@@ -33,4 +33,8 @@ export enum AttributeNames {
   // PG specific -- not specified by spec
   PG_VALUES = 'pg.values',
   PG_PLAN = 'pg.plan',
+
+  // PG-POOL specific -- not specified by spec
+  IDLE_TIMEOUT_MILLIS = 'idle.timeout.millis',
```

Comment:
why add them here ?

Reply:
??? where else do you think it should be added ?

Reply:
Well in the instrumentation package ? You added it into the plugin

Reply:
whoops, thx for spotting